roles/idrac_user/tasks/get_user.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: Set user_name based on new_user_name
+  ansible.builtin.set_fact:
+    user_name: "{{ new_user_name if new_user_name is defined else user_name }}"
```

Comment:
change this fact name to use the format idrac_user_present_name

Reply:
done